src/rpdk/core/test.py

```diff
@@ -52,14 +54,46 @@ def temporary_ini_file():
         yield str(path)
 
 
-def get_overrides(root):
+def get_cloudformation_exports(region_name, endpoint_url):
+    session = create_sdk_session(region_name)
+    cfn_client = session.client("cloudformation", endpoint_url=endpoint_url)
+    paginator = cfn_client.get_paginator("list_exports")
+    pages = paginator.paginate()
+    exports = {}
+    for page in pages:
+        exports.update({export["Name"]: export["Value"] for export in page["Exports"]})
```

Comment:
so we pull all the exports for an account every time we run cfn-test? there doesn't appear to be any limit on the number of exports for an account https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/cloudformation-limits.html

Reply:
Max number of exports should be 12000. Outputs per stack is 60 and number of stacks is 200. Though I'm assuming outputs behave like parameters and nested stack outputs all count towards the same limit. 

Improving the best case (we can't improve the worst case) will increase the time complexity as well.

Reply:
Seems okay since this function shouldn't get called unless opting into this feature

Should we catch exceptions like [`ExpiredToken`](https://github.com/aws-cloudformation/cloudformation-cli/issues/147#issuecomment-664663330)?